au/code/au/units/test/football_fields_test.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright 2025 Aurora Operations, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "au/units/football_fields.hh"
+
+#include "au/testing.hh"
+#include "au/units/meters.hh"
+#include "au/units/yards.hh"
+#include "gmock/gmock.h"
+#include "gtest/gtest.h"
+
+namespace au {
+
+using ::testing::Eq;
+using ::testing::Lt;
+
+TEST(FootballFields, HasExpectedLabel) { expect_label<FootballFields>("ftbl_fld"); }
+
+TEST(FootballFields, HasCorrectQuantityRelationshipWithYards) {
+    EXPECT_THAT(football_fields(1), Eq(yards(100)));
+}
+
+TEST(FootballFields, FourFootballFieldsIsLessThanKnownFirstlightLidarRange) {
+    // Sources:
+    // https://blog.aurora.tech/progress/firstlight-lidar-on-a-chip
+    // https://ir.aurora.tech/news-events/press-releases/detail/119/aurora-begins-commercial-driverless-trucking-in-texas
+    EXPECT_THAT(football_fields(4), Lt(meters(400)));
```

Comment:
Nit: This reads a bit backwards to me.  Since we're really trying to an "kxcd" unit test and verify what we say in our press release, I'm thinking something like:

```suggestion
    // https://blog.aurora.tech/progress/firstlight-lidar-on-a-chip
    constexpr auto lidar_range_minimum = meters(400);

    // https://ir.aurora.tech/news-events/press-releases/detail/119/aurora-begins-commercial-driverless-trucking-in-texas
    EXPECT_THAT(lidar_range_minimum, Gt(football_fields(4));
```

Reply:
There appears to have been a strange character introduced somhow in the new `EXPECT_THAT` line.  I got really bizarre compiler errors:

```
ERROR: /home/chogg/au/au/BUILD.bazel:171:8: Compiling au/code/au/units/test/football_fields_test.cc failed: (Exit 1): cc_wrapper.sh failed: error executing command (from target //au:units_test) external/llvm_14_toolchain/bin/cc_wrapper.sh -U_FORTIFY_SOURCE '--target=x86_64-unknown-linux-gnu' -U_FORTIFY_SOURCE -fstack-protector -fno-omit-frame-pointer -fcolor-diagnostics -Wall -Wshadow ... (remaining 54 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
au/code/au/units/test/football_fields_test.cc:39:5: error: unterminated function-like macro invocation
    EXPECT_THAT(lidar_range_minimum, Gt(football_fields(4));
    ^
external/com_google_googletest/googlemock/include/gmock/gmock-matchers.h:5435:9: note: macro 'EXPECT_THAT' defined here
#define EXPECT_THAT(value, matcher) \
        ^
au/code/au/units/test/football_fields_test.cc:47:19: error: expected '}'
}  // namespace au
                  ^
au/code/au/units/test/football_fields_test.cc:34:66: note: to match this '{'
TEST(FootballFields, KnownLidarRangeIsAtLeastFourFootballFields) {
                                                                 ^
au/code/au/units/test/football_fields_test.cc:47:19: error: expected '}'
}  // namespace au
                  ^
au/code/au/units/test/football_fields_test.cc:23:14: note: to match this '{'
namespace au {
             ^
3 errors generated.
Target //au:units_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 0.970s, Critical Path: 0.84s
INFO: 2 processes: 2 internal.
FAILED: Build did NOT complete successfully
//au:units_test                                                 FAILED TO BUILD
```

Ultimately, I had to delete and retype the line to get it to build.  I have no idea what happened... strange!

No big deal as I needed to update the `using` statement for `Gt` anyway.

Reply:
> There appears to have been a strange character introduced somhow in the new EXPECT_THAT line.

Rather, a rather obvious character missing at the end.  It was missing a `)`.  I run into this all the time as I often forget how many closing braces I have pending at the end of one of these lines... 🤣